### PR TITLE
Make the member images on the page appear dynamically. #138

### PR DESCRIPTION
**Pull request by rahilkapoor**
Fixed for Mobile Version

mob

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,9 +6,9 @@
     <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
     <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
     <title>Real Dev Squad</title>
-    <link rel="stylesheet" href="./css/main.css" />
-    <link rel="stylesheet" href="./css/animate.css" />
-    <link rel="stylesheet" href="./css/navbar.css" />
+    <link rel="stylesheet" href="./css/main.css" defer />
```
Comment (harshith-venkatesh) [inline index.html:9]: defer is used for scripts https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script

File: index.html [hunk 2]
```diff
@@ -6,9 +6,9 @@
...
+    <link rel="stylesheet" href="./css/animate.css" defer />
```
Comment (harshith-venkatesh) [inline index.html:10]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Link_prefetching_FAQ

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -319,13 +319,7 @@ <h2 class="title-bullet-points">Soft Skills</h2>
...
         </section>
 
-        <section class="member-imgs" id="members">
-          <img class="member-img member_animation" alt="member-img" />
-          <img class="member-img member_animation" alt="member-img" />
-          <img class="member-img member_animation" alt="member-img" />
-          <img class="member-img member_animation" alt="member-img" />
-          <img class="member-img member_animation" alt="member-img" />
-        </section>
+        <section class="member-imgs" id="members"></section>
```
Comment (swarajpure) [inline index.html:322]: Empty tags can be self closing
`<section class=.......... />`

File: js/main.js [hunk 4]
```diff
@@ -28,8 +28,17 @@ const selectRandom = (memberImgArr, n) => {
   return result;
 };
 
+const numOfMembers = 5;
+let memberSection = document.getElementById('members');
+for (let i = 0; i < numOfMembers; i++) {
+  const memberTags = document.createElement('img');
+  memberTags.classList.add('member-img', 'member_animation');
+  memberTags.setAttribute('alt', 'member-img');
```
Comment (swarajpure) [inline js/main.js:36]: `Alt` text could non-technical? It should be simple english, if the images fail to load, hovering upon them we should see that text right?

**Review by swarajpure — CHANGES_REQUESTED** [review]
Thanks!
Requesting some minor changes, please check!

File: index.html (hunk 3 above)
Comment (rahilkapoor) [reply index.html:322]: We are dynamically adding tags later...So we need the closing tag here!

File: js/animate.js [hunk 5]
```diff
@@ -8,17 +8,28 @@ const randomNumberWithinRange = (min, max) => {
...
+  const deg = n * ((360 / numOfMembers) * (Math.PI / 180));
   return 160 * Math.sin(deg);
 };
 
 const memberList = document.querySelectorAll('.member_animation');
 
+const handleIntersection = (entries) => {
+  entries.forEach(({ isIntersecting }) => {
+    if (isIntersecting) addClassName();
+    else removeClassName();
```
Comment (swarajpure) [inline js/animate.js:25]: How about?
```JS
isIntersecting ? addClassName() : removeClassName()
```

Comment (rahilkapoor) [reply js/animate.js]: This was a really good one! Thanks, Swaraj!

**Review by swarajpure — APPROVED** [review]
Looks good to me! 🔥

Approving from my side

**Review by Vividh25 — APPROVED** [review]
Great work here!

**Pull request merged**